low-level/solution/lr2/server/manifest.py

```diff
@@ -95,6 +96,18 @@ def __init__(self):
         }
         self.service = Service()
         self.platform = Platform()
+        # Resources and their common key path.
+        self.manifest_health_index_map_key = {
+            "hw": {
+                "disk": '["serial_number"]',
+                "psu": '["serial_number"]',
+                "nw_port": '["logical_name"]'
```

Comment:
ok for nw_port now its not matching with that of health index map key, it needs to have common. This table actually should be in resource map, from where health & manifest can use